src/span.js

```diff
@@ -106,23 +206,6 @@ export default class Span {
         return this;
     }
 
-    /**
-     * Set an arbitrary key-value string pair that will be carried along the
-     * full path of a trace.
-     *
-     * All spans created as children of this span will inherit the baggage items
-     * of this span.
-     *
-     * Baggage items are copied between all spans, both in-process and across
-     * distributed requests, therefore this feature should be used with care to
-     * ensure undue overhead is not incurred.
-     *
-     * Keys are case insensitive and must match the regular expresssion
-     * `[a-z0-9][-a-z0-9]*`.
-     *
-     * @param {string} key
-     * @param {string} value
-     */
```

Comment:
These comments likely should not be removed as that means these methods will no longer show up in the generated API docs.